### ISSUE TITLE
PWA Accessibility Check

### ISSUE THREAD

**Issue by oninross** [kenwheeler/slick#2937]
I scanned one of my projects through Lighthouse, a Progressive Web App checker, and it gave me these issues.
![screenshot_061617_075233_am](https://user-images.githubusercontent.com/2233813/27203340-c2979f80-5268-11e7-81ff-a6cdada5fc27.jpg)

It would be nice if the plugin will pass Lighthouse.  Its a great plugin.

**Comment by eaarranz** [comment]
Yes, we are facing the same problem, it would be nice to fit the Lighthouse requirements.
